docs/examples/merging/example02-field-tender.json

```diff
@@ -0,0 +1,84 @@
+{
+  "uri": "https://standard.open-contracting.org/examples/releases/ocds-k50g02-13-9-368828-tender.json",
+  "publisher": {
+    "name": "Colombia Compra Eficiente"
+  },
+  "publishedDate": "2013-07-28T09:40:10.000Z",
```

Comment:
Think there's a typo here on the datetime

Reply:
I'm not sure it's me but I can't see anything wrong with the date!

Reply:
Looks okay – milliseconds are allowed.